packages/govuk-frontend-review/src/app.mjs

```diff
@@ -80,10 +80,81 @@ export default async () => {
   // Configure nunjucks
   const env = nunjucks.renderer(app)
 
+  // Define parameters
+
+  /**
+   * Handle parameter :componentName
+   *
+   * Finds all component fixtures and default example
+   */
+  app.param(
+    'componentName',
```

Comment:
I've added Express.js `app.param()` to follow up code review feedback in https://github.com/alphagov/govuk-frontend/pull/4424#discussion_r1385127501